doc/help-scoring.md

```diff
@@ -74,31 +76,54 @@ This category measures if a package has documentation in two areas:
   - At least 20% of the public API members contain [API
     documentation](https://dart.dev/guides/libraries/create-library-packages#documenting-a-library).
 
-### Support multiple platforms
+### Platform support
 
 Packages are encouraged to support multiple platforms, to enable app developers
 to support a wide variety of platforms for their apps. This includes Dart's
 [native and web](https://dart.dev/platforms) platforms, and Flutter's
 [mobile](https://flutter.dev/docs), [web](https://flutter.dev/web), and
 [desktop](https://flutter.dev/desktop) targets.
 
-Pub.dev uses two different algorithms for determining which platforms are supported:
+pub.dev knows about the following platforms:
 
- - **Flutter packages**: For packages that depend on the Flutter SDK, platform
-   support is determined by looking at the [`platforms`
-   tag](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms)
-   in the `pubspec.yaml` file.
+* windows
```

Comment:
I think the proper capitalisation for these is:

Windows
Linux
macOS
Android
iOS
Web

Reply:
Yeah - I kind-a wanted to give the keywords one could put in the pubspec, but that should not go here.

Fixed.